stdlib/xml/dom/expatbuilder.pyi

```diff
@@ -1,3 +1,102 @@
-from typing import Any
+from typing import Any, NoReturn, Union
+from typing_extensions import Literal
+from xml.dom.minidom import Document, DOMImplementation, Node, TypeInfo
+from xml.dom.xmlbuilder import DOMBuilderFilter, Options
 
-def __getattr__(name: str) -> Any: ...  # incomplete
+TEXT_NODE = Node.TEXT_NODE
+CDATA_SECTION_NODE = Node.CDATA_SECTION_NODE
+DOCUMENT_NODE = Node.DOCUMENT_NODE
+FILTER_ACCEPT = DOMBuilderFilter.FILTER_ACCEPT
+FILTER_REJECT = DOMBuilderFilter.FILTER_REJECT
+FILTER_SKIP = DOMBuilderFilter.FILTER_SKIP
+FILTER_INTERRUPT = DOMBuilderFilter.FILTER_INTERRUPT
+theDOMImplementation: DOMImplementation | None
+
+class ElementInfo:
+    tagName: Any
+    def __init__(self, tagName: Any, model: Any | None = ...) -> None: ...
```

Comment:
Arguments with unknown types are best left unannotated. However, this doesn't apply to attributes (e.g. `tagName`), and we also accept `Any | None` when the default value is `None`, because that's what mypy's stubgen does.

```suggestion
    def __init__(self, tagName, model: Any | None = ...) -> None: ...
```

Speaking of stubgen, consider giving it a try if you haven't tried it yet :)